managed/ui/src/components/config/Storage/StorageConfigurationContainer.js

```diff
@@ -31,6 +32,21 @@ const mapDispatchToProps = (dispatch) => {
   return {
     addCustomerConfig: (config) => {
       return dispatch(addCustomerConfig(config)).then((response) => {
+        if (response.error) {
+          const errorMessageObject = response.payload?.response?.data?.error || response.payload.message;
+          Object.keys(errorMessageObject).forEach((errorKey) => {
+            toast.error(
+              <ol>
```

Comment:
Do we really need to do it in this way? And Can you please add a screenshot of this as well?

Reply:
Yeah, it is having a list of error for any single field